Deploy-CAM.ps1

```diff
@@ -2519,6 +2519,54 @@ function Confirm-ModuleVersion()
     return $true
 }
 
+# Create a custom role for CAM with necessary permissions
+# Currently it is the based off of the Contributor Role excpept we remove Storage Access and VM Disk Blob Read Access
+# Use 'Get-AzureRmProviderOperation *' to get a list of Azure Operations and their details
+# See https://docs.microsoft.com/en-us/azure/active-directory/role-based-access-built-in-roles for details on Azure Built in Roles
+function Get-CAMRoleDefinition() {
+    param(
+        [parameter(Mandatory = $false)]
+        [String]$subscriptionId
+    )
+
+    $camCustomRoleDefinition = Get-AzureRmRoleDefinition "Cloud Access Manager"
```

Comment:
There are 6 occurrences of the role string "Cloud Access manager" in the code, which could be single sourced but your call.

Reply:
good point